### PR TITLE
ci: add unit and visual tests

### PR DESCRIPTION
**Pull request by juliodialpad**
## Description

- Added Unit and Visual tests.
- General cleanup of unneeded files.
- Removed message_input from index exports
- Removed emoji_picker from index exports
- Added rich_text_editor to message_input exports

## Pull Request Checklist

 - [x] Ask the contributors if a similar effort is already in process or has been solved.
 - [x] Review the [contribution guidelines](https://github.com/dialpad/dialtone/blob/staging/.github/CONTRIBUTING.md).
 - [x] Use `staging` as your pull request's base branch. (All PRs using `production` as its base branch will be declined).
 - [x] Ensure all `gulp` scripts successfully compile.
 - [ ] Update, remove, or extend all affected documentation.
 - [ ] Ensure no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).

### CODE REVIEW

**Review by iropolo — APPROVED** [review]

File: apps/dialtone-vue3-documentation/package.json [hunk 1]
```diff
@@ -3,14 +3,14 @@
   "version": "1.0.0",
   "description": "Dialtone Vue library documentation site",
   "scripts": {
-    "build:documentation:functions": "npx typedoc --plugin typedoc-plugin-markdown",
```
Comment (braddialpad) [inline apps/dialtone-vue3-documentation/package.json:6]: Don't we still need to run this for this? https://vue.dialpad.design/?path=/docs/functions-date-and-time--docs

File: apps/dialtone-vue3-documentation/package.json [hunk 2]
```diff
@@ -3,14 +3,14 @@
...
-    "lint:staged": "markdownlint --fix",
+    "lint:fix": "pnpm run lint --fix",
     "lint": "markdownlint '**/*.mdx'",
     "start": "STORYBOOK_ENV=development storybook dev -p 9010",
     "clean": "rm -rf ./storybook-static",
-    "build": "npm run build:documentation:functions && npm run clean && STORYBOOK_ENV=production storybook build && touch ./storybook-static/.nojekyll",
+    "build": "pnpm run clean && STORYBOOK_ENV=production storybook build && touch ./storybook-static/.nojekyll",
     "build:docs-only": "STORYBOOK_ENV=production && build:documentation:functions && storybook build --docs",
```
Comment (braddialpad) [inline apps/dialtone-vue3-documentation/package.json]: `build:documentation:functions` is still in this line

File: apps/dialtone-vue3-documentation/package.json (hunk 1 above)
Comment (juliodialpad) [reply apps/dialtone-vue3-documentation/package.json:6]: Yes, but it wasn't working correctly, will try to fix it

File: apps/dialtone-vue3-documentation/package.json (hunk 2 above)
Comment (juliodialpad) [reply apps/dialtone-vue3-documentation/package.json]: ~will add it back once it's working~

Oh you mean on line 11, I didn't see that. I'll fix it and add it back to the scripts

File: apps/dialtone-vue3-documentation/package.json (hunk 1 above)
Comment (juliodialpad) [reply apps/dialtone-vue3-documentation/package.json:6]: Fixed and added back

File: apps/dialtone-vue3-documentation/package.json (hunk 2 above)
Comment (juliodialpad) [reply apps/dialtone-vue3-documentation/package.json]: added back

**Review by braddialpad — APPROVED** [review]

**Pull request merged**